core/src/main/java/org/apache/accumulo/core/client/admin/InstanceOperations.java

```diff
@@ -149,6 +150,46 @@ boolean testClassLoad(final String className, final String asTypeName)
    */
   void waitForBalance() throws AccumuloException;
 
+  /**
+   * Throws an exception if a tablet server can not be contacted.
+   *
+   * @param txids
+   *          Transaction IDs.
+   * @since 2.1.0
+   */
+  void fateFail(List<String> txids) throws AccumuloException;
+
+  /**
+   * Throws an exception if a tablet server can not be contacted.
+   *
+   * @param txids
+   *          Transaction IDs.
+   * @since 2.1.0
+   */
+  void fateDelete(List<String> txids) throws AccumuloException;
+
+  /**
+   * Throws an exception if a tablet server can not be contacted.
+   *
+   * @param txids
+   *          Transaction IDs.
+   * @param tStatus
+   *          Parsed TStatus for print filter.
+   * @return String containing the output to print to the shell.
+   * @since 2.1.0
+   */
+  String fatePrint(List<String> txids, List<String> tStatus) throws AccumuloException;
+
+  /**
+   * Throws an exception if a tablet server can not be contacted.
+   *
+   * @param txids
+   *          Transaction IDs.
+   * @return String containing the output to print to the shell.
+   * @since 2.1.0
+   */
+  String fateDump(List<String> txids) throws AccumuloException;
```

Comment:
See previous review conversation for discussion about consolidating these, and having a meaningful return type other than String. These APIs should not be written exclusively for use with the Shell, but should have meaningful return types of their own, and not just formatted strings for the shell.